buildkite/bazelci.py

```diff
@@ -2148,7 +2148,10 @@ def expand_test_target_patterns(bazel_binary, test_targets):
         + [
             "--nosystem_rc",
             "--nohome_rc",
-            "query",
+            "cquery",
+            "--output=starlark",
+            # Print out the label only if the test is compatible.
+            "--starlark:expr=target.label if 'IncompatiblePlatformProvider' not in providers(target) else ''",
```

Comment:
Is there a way to validate that I did this correctly? I used this query in the past, but it would be lovely if there was an integration test for this file or something.